api/src/opentrons/protocol_engine/commands/move_labware.py

```diff
@@ -94,12 +97,29 @@ async def execute(self, params: MoveLabwareParams) -> MoveLabwareResult:
         """Move a loaded labware to a new location."""
         # Allow propagation of LabwareNotLoadedError.
         current_labware = self._state_view.labware.get(labware_id=params.labwareId)
+        current_labware_definition = self._state_view.labware.get_definition(
+            labware_id=params.labwareId
+        )
         definition_uri = current_labware.definitionUri
 
         empty_new_location = self._state_view.geometry.ensure_location_not_occupied(
```

Comment:
sorry for the nip but it can also be a OnLabwareLocation correct? can we change the var name to `move_location` or something like that?
```suggestion
        empty_new_location = self._state_view.geometry.ensure_location_not_occupied(
```